### DISCUSSION TITLE
Azure deployment fails with Oqtane 2.2

### DISCUSSION THREAD

**Discussion by mixlplix** [oqtane/oqtane.framework#1644] [General] [answered]
I have tried multiple times to deploy Oqtane to Azure using the link on then github readme file.  This is the error I get.  Any ideas?

![image](https://user-images.githubusercontent.com/1314264/132104362-e21f8b48-a3e3-47d9-97e7-2a0a8f7b1ac9.png)

**Comment by mixlplix** [comment]
Here are the details of the error.  Not much help:

{"code":"DeploymentFailed","message":"At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details.","details":[{"code":"Conflict","message":"{\r\n \"status\": \"Failed\",\r\n \"error\": {\r\n \"code\": \"ResourceDeploymentFailure\",\r\n \"message\": \"The resource operation completed with terminal provisioning state 'Failed'.\"\r\n }\r\n}"}]}

**Comment by hishamco** [comment]
@ADefWebserver any idea?

**Comment by ADefWebserver** [comment]
To know the exact cause these directions would need to be followed: https://aka.ms/DeployOperations
Or, Try this method instead: [Installing Blazor Oqtane](https://blazorhelpwebsite.com/ViewBlogPost/1)

**Reply by markdav-is** [reply]
@mixlplix please mark this as the answer

**Comment by mixlplix** [comment, marked as answer]
Thank you, that worked.
